src/containers/StudyEdit/StudyEditForm.tsx

```diff
@@ -0,0 +1,127 @@
+  StudyDescriptionWrapper,
+} from './style';
+
+function StudyEditForm() {
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imageSrc, setImageSrc] = useState<File | null>(null);
+  const [thumbnailImage, setThumbnailImage] = useState<string>('');
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const data = await fetchStudyDetails();
+      const { title, description, imageUrl } = data;
+
+      setTitle(title);
+      setDescription(description);
+      setThumbnailImage(imageUrl);
+    };
+
+    fetchData();
+  }, []);
+
+  const encodeFile = (fileBlob: File) => {
+    const reader = new FileReader();
+    if (!fileBlob) return;
+    reader.readAsDataURL(fileBlob);
+
+    return new Promise<void>((resolve) => {
+      reader.onload = () => {
+        const result = reader.result as string;
+        setThumbnailImage(result);
+
+        resolve();
+      };
+    });
+  };
+
+  const onImageChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
+      const maxSize = 1 * 1024 * 1024;
+      const fileSize = e.target.files[0].size;
+
+      if (fileSize > maxSize) {
+        alert('첨부 파일은 최대 1MB 입니다.');
+        return;
+      }
+      setImageSrc(e.target.files[0]);
+      encodeFile(e.target.files[0]);
+    }
+  };
+
+  const onSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    console.log('제목 ' + title);
+    console.log('이미지 ' + imageSrc);
+    console.log('내용' + description);
+  };
+  return (
+    <form onSubmit={onSubmit}>
+      <StudyEditWrapper>
+        <StudyEditHeading>
+          <Typography variant='h4'>스터디 수정</Typography>
+          <LabelInput
+            id='study-title'
+            label='스터디명'
+            value={title}
+            fullWidth={true}
+            onChange={(e) => {
+              setTitle(e.target.value);
+            }}
+          />
+        </StudyEditHeading>
+        <StudyEditImageWrapper>
+          <Typography variant='h5'>대표 이미지</Typography>
+          <ImageWrapper>
+            {thumbnailImage && (
+              <img
+                src={thumbnailImage}
+                alt='study-image'
+                style={{
+                  width: '100%',
+                  height: '100%',
+                  borderRadius: '0.5rem',
+                  objectFit: 'cover',
+                }}
+              />
+            )}
+          </ImageWrapper>
+          <ButtonWrapper>
+            <FileInput message='이미지 업로드' onChange={onImageChange} />
+          </ButtonWrapper>
+        </StudyEditImageWrapper>
+        <StudyDescriptionWrapper>
+          <Typography variant='h5'>상세 설명</Typography>
+          <MultiLineInput
+            id='study-description'
+            placeholder='스터디 내용을 기재해주세요.'
+            value={description}
+            height='600px'
+            onChange={(e) => {
+              setDescription(e.target.value);
+            }}
```

Comment:
```suggestion
     const handleDesription = (e: React.ChangeEvent<HTMLInputElement>) => {
              const {value=''} = e.target;
              setDescription(value);
            }

            onChange={handleDesription}
```
위처럼 콜백함수로 빼서 정의하시는것도 좋아보입니다

Reply:
title, description 모두 handler 함수로 분리해 적용했습니다. 감사합니다 😁

c7b0c0cbe068e362f6e6e916e01271a0ca6f5756